uint8 id
string kinematic_type
uint8[] motor_ids